client/src/theme/_fonts.scss

```diff
@@ -3,14 +3,126 @@
 
 // Do your @font-face embedding/include mixins here, in the privacy of your own partial
 
+// TypeKit fallback serif font
+// --------------------------
+@font-face {
+  font-family: 'aleo';
+  src: url('assets/fonts/aleo/aleo-regular/Aleo-Regular.eot');
+  src: url('assets/fonts/aleo/aleo-regular/Aleo-Regular.eot?#iefix') format('embedded-opentype'),
+  url('assets/fonts/aleo/aleo-regular/Aleo-Regular.woff') format('woff'),
+  url('assets/fonts/aleo/aleo-regular/Aleo-Regular.ttf') format('truetype'),
+  url('assets/fonts/aleo/aleo-regular/Aleo-Regular.svg#aleoregular') format('svg');
+
+  font-weight: $regular;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'aleo';
+  src: url('assets/fonts/aleo/aleo-italic/Aleo-Italic.eot');
+  src: url('assets/fonts/aleo/aleo-italic/Aleo-Italic.eot?#iefix') format('embedded-opentype'),
+  url('assets/fonts/aleo/aleo-italic/Aleo-Italic.woff') format('woff'),
+  url('assets/fonts/aleo/aleo-italic/Aleo-Italic.ttf') format('truetype'),
+  url('assets/fonts/aleo/aleo-italic/Aleo-Italic.svg#aleoregular') format('svg');
+
+  font-weight: $regular;
+  font-style: italic;
+}
+
+@font-face {
+  font-family: 'aleo';
+  src: url('assets/fonts/aleo/aleo-light/Aleo-Light.eot');
+  src: url('assets/fonts/aleo/aleo-light/Aleo-Light.eot?#iefix') format('embedded-opentype'),
+  url('assets/fonts/aleo/aleo-light/Aleo-Light.woff') format('woff'),
+  url('assets/fonts/aleo/aleo-light/Aleo-Light.ttf') format('truetype'),
+  url('assets/fonts/aleo/aleo-light/Aleo-Light.svg#aleoregular') format('svg');
+
+  font-weight: $light;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'aleo';
+  src: url('assets/fonts/aleo/aleo-bold/Aleo-Bold.eot');
+  src: url('assets/fonts/aleo/aleo-bold/Aleo-Bold.eot?#iefix') format('embedded-opentype'),
+  url('assets/fonts/aleo/aleo-bold/Aleo-Bold.woff') format('woff'),
+  url('assets/fonts/aleo/aleo-bold/Aleo-Bold.ttf') format('truetype'),
+  url('assets/fonts/aleo/aleo-bold/Aleo-Bold.svg#aleoregular') format('svg');
+
+  font-weight: $bold;
+  font-style: normal;
+}
+
+// TypeKit fallback san-serif font
```

Comment:
This commit looks great!! This is a super nit-picky thing, but I think CIC style is to spell it `sans-serif.` Since it's a comment I think this can still go in, though!